contribute/style-guides/documentation-markdown-guide.md

```diff
@@ -0,0 +1,147 @@
+# Markdown style guide
+
+## Headers
+
+In Markdown, the number of ``#`` symbols creates different heading levels, similar to
+HTML heading levels.
+
+**Example**
+* \# is \<h1>.
+* \#\# is \<h2>.
+* \#\#\# is \<h3>.
+
+Start your document with a single ``#`` for the title of the page. Add the sub-headings with two ``##``.
+
+## Bold and emphasis
+
+* Make text **bold** using two asterisks.
+
+**Example** It is ``**important**`` to use Github Flavored Markdown emoji consistently.
+
+* Make text ``*emphasized*`` using single `` _underscores_`` or a single asterisk.
+
+**Example** Github Flavored Markdown emoji should _only_ appear in specific cases.
+
+
+## Handling punctuation
+
+When writing and marking up text with bold or emphasis markdown, include any
+punctuation close to the target text within the markdown:
+
+**Example** **TODO:**, not **TODO**:.
+
+**Example** The following emoji are considered _acceptable:_ Radio buttons, tick marks, and heavy plus signs.
+
+**Example** Emoji became popular in the **mid-2000's**
+
+## Links and References
```

Comment:
Per the documentation style guide, all headings should be in sentence case. Initial cap only.